test/e2e-tests.sh

```diff
@@ -111,4 +111,42 @@ run_test  "show logs" tkn taskrun logs test-template-volume
 run_test  "list pipelineresources" tkn resource list
 run_test  "describe pipelineresource" tkn resource desc skaffold-git
 
+# delete pipeline, pipelinerun, task, taskrun, and pipelineresource
+run_test  "delete pipeline" expect -c "
+  spawn  tkn pipeline delete output-pipeline
```

Comment:
Is it better to be more simple using ```-f``` flag??

Reply:
I think it's fine with -f but doing with -f is extra better , you can perhaps move a lot of it to a generic shell function to simplify it,

Reply:
> you can perhaps move a lot of it to a generic shell function to simplify it,

thanks!! I'll try it.

Reply:
Hmm... Test conainter image seems not to have expect command so I fix to use ```-f``` flag for now.
I think we can choose another way to this like below, any ideas?

1. find a way use another command that has the same functionality as the expect command. (but I have no idea for now sorry :sweat_smile:)
2. Include expect command in test container image (Could it?)